src/editors/containers/VideoEditor/components/VideoSettingsModal/components/LicenseWidget.jsx

```diff
@@ -5,10 +5,18 @@ import { useDispatch } from 'react-redux';
 import hooks from './hooks';
 import CollapsibleFormWidget from './CollapsibleFormWidget';
 
+/**
+ * Collapsible Form widget controlling videe licence type and details
+ */
 export const LicenseWidget = () => {
   const dispatch = useDispatch();
-  const licenseType = hooks.widgetValue(hooks.selectorKeys.licenseType, dispatch);
-  const licenseDetails = hooks.widgetValue(hooks.selectorKeys.licenseDetails, dispatch);
+  const { licenseType, licenseDetails } = hooks.widgetValues({
+    dispatch,
+    fields: {
+      [hooks.selectorKeys.licenseType]: hooks.genericWidget,
```

Comment:
Curious: what caused these to be two separate widgets?

Reply:
Mostly because they are 2 separate data fields, with different shapes

Reply:
Because one is an object, and I don't want an embedded object if I can avoid it.  Makes sense for each to be an individual state field, and they are easy to control that way.